pandas/tests/test_panel.py

```diff
@@ -1560,6 +1560,17 @@ def test_truncate_fillna_bug(self):
         # it works!
         result.fillna(value=0.0)
 
+    def test_truncate_outofbounds(self):
```

Comment:
you said done, but it doesn't appear to be pushed. you need to use ``_construct``